Makefile

```diff
@@ -142,16 +139,32 @@ clean_integration:
 ##########
 ## HOST ##
 ##########
-build_hosts: build_host_js
+build_hosts: build_host_js build_host_py
 clean_hosts:
 	rm -rf ${HOST_JS_ASSETS}
 	cd host/js && yarn clean
+	rm -rf ${HOST_PY_ASSETS}
 
 # copy wasm always because cached docker artifacts can have older timestamp
 build_host_js: ${CORE_ASYNCIFY_WASM}
 	mkdir -p ${HOST_JS_ASSETS}
-	cp ${CORE_ASYNCIFY_WASM} ${HOST_JS_ASSETS_WASM_CORE}
+	cp ${CORE_ASYNCIFY_WASM} ${HOST_JS_ASSETS}/core-async.wasm
 	cd host/js && yarn install && yarn build	
 
-test_host_js:
+test_host_js: build_host_js ${TEST_CORE_ASYNCIFY_WASM}
+	cp ${TEST_CORE_ASYNCIFY_WASM} ${HOST_JS_ASSETS}/test-core-async.wasm
 	cd host/js && yarn test
+
+deps_host_py:
+	cd host/python; test -d venv || python3 -m venv venv; source venv/bin/activate; \
+	python3 -m pip install -r requirements.txt
```

Comment:
Is this step just to install dependencies? If so, pyproject.toml should be used.

https://github.com/pypa/packaging.python.org/issues/685#issuecomment-1321616748

Reply:
To be honest so far I've understood pretty much nothing about the pyproject.toml file, especially because somehow there is no one tool to manage python packages and I don't even know which one to pick, so I did what I know and set up a virtualenv.

I was hoping we could handle the packaging in a separate PR as that technically doesn't block testing the Python host.

Reply:
Well it's not packaging, but dependency management. But understand what you say. I will handle it.